modules/kafka/src/main/java/org.testcontainers.containers/KafkaContainer.java

```diff
@@ -0,0 +1,80 @@
+package org.testcontainers.containers;
+
+import org.testcontainers.utility.Base58;
+
+import java.util.stream.Stream;
+
+public class KafkaContainer extends GenericContainer<KafkaContainer> {
+
+    public static final int KAFKA_PORT = 9092;
+
+    public static final int ZOOKEEPER_PORT = 2181;
+
+    protected String externalZookeeperConnect = null;
+
+    protected SocatContainer proxy;
+
+    public KafkaContainer() {
+        this("4.0.0");
+    }
+
+    public KafkaContainer(String confluentPlatformVersion) {
```

Comment:
Maybe I missed something, but don't we need a custom wait strategy? Or is port based wait strategy already enough?

Reply:
yes, Kafka is ready to be used when the port is open

Reply:
I at least had some instances, where I got the following error when creating a topic:
`ERROR org.apache.kafka.common.errors.InvalidReplicationFactorException: replication factor: 1 larger than available brokers: 0
 (kafka.admin.TopicCommand$)`.

This did not happen for me, when I used 
`.waitingFor(new LogMessageWaitStrategy().withRegEx(/(?s).*started \(kafka\.server\.KafkaServer\).*/)) `

Reply:
"Started Kafka" != "Initialized Kafka", your clients should be able to sync the cluster state before doing operations (and most of them do).

Waiting for cluster initialization instead of just startup will just make the tests slower while it should be handled by your consumers / producers

Reply:
I see, in this case we were using the bundled helper scripts, like `kafka-topics.sh` (but this is mostly a rudiment from having to use kafka with `GenericContainer`).